src/spikeinterface/preprocessing/filter_gaussian.py

```diff
@@ -95,16 +95,14 @@ def _create_gaussian(self, N: int, cutoff_f: float):
         sf = self.parent_recording_segment.sampling_frequency
         faxis = np.fft.fftfreq(N, d=1 / sf)
 
-        from scipy.stats import norm
-
         if cutoff_f > sf / 8:  # The Fourier transform of a Gaussian with a very low sigma isn't a Gaussian.
             sigma = sf / (2 * np.pi * cutoff_f)
-            limit = int(round(6 * sigma)) + 1
+            limit = int(round(5 * sigma)) + 1
```

Comment:
It's a cutoff,
Since computing the Gaussian kernel is slow, you don't want to spend a lot of time computing it for values that are going to end up being 0 anyway.

I thought about using `margin_sd`, but it's not really about margins on the chunks. Here it's the cutoff for the kernel ...

Reply:
got it, maybe exposing it as an argument then? I know it's very low-level, but I don't like hardcoded stuff..up to you though!

Reply:
I mean we could,
I really don't see the point though, and I don't know how I'd name it.
But if you prefer it that way I can implement it :)